FESS-GUI/mainwindow.h

```diff
@@ -40,6 +41,7 @@ class MainWindow : public QMainWindow
     ~MainWindow();
 
 private:
+    CommonDeviceInterface *flywheelCommunicationDevice;
```

Comment:
Thanks for commenting this. I actually had a thought about this after our meeting. As far as switching between the two, I think the main window should have two. It should initialize the flywheelOperation with one, then if he user switches them, call flywheelOperation.switchDevice(device). 

As far as modifying and sharing in the future, I just prefer to keep the amount of dependencies as low as possible at any moment in time. But sure, if we plan to let the main window have two in the future, I won't protest keeping it in main window for the time being.